File: 4-go/dtos/order.go
package dtos

type Order struct {
	ID           int64   `json:"id"`
	IdUser       int64   `json:"idUser"`
	Amount       float32 `json:"price"`
	PurchaseDate string  `json:"purchaseDate"`
	DeliveryType int     `json:"deliveryType"`
	PaymentType  int     `json:"paymentType"`
}

File: 5-react/my-app/src/models/order.js
export class Order {
    id;
    idUser;
    amount;
    purchaseDate;
    deliveryType;
    paymentType;

    constructor(id, idUser, amount, purchaseDate, deliveryType, paymentType) {
        this.id = id;
        this.idUser = idUser;
        this.amount = amount;
        this.purchaseDate = purchaseDate;
        this.deliveryType = deliveryType;
        this.paymentType = paymentType;
    }
}